source/opt/wrap_opkill.cpp

```diff
@@ -108,13 +115,22 @@ uint32_t WrapOpKill::GetVoidFunctionTypeId() {
   return type_mgr->GetTypeInstruction(&func_type);
 }
 
-uint32_t WrapOpKill::GetOpKillFuncId() {
-  if (opkill_function_ != nullptr) {
-    return opkill_function_->result_id();
+uint32_t WrapOpKill::GetKillingFuncId(SpvOp opcode) {
+  //  Parameterize by opcode
+  assert(opcode == SpvOpKill
+
+         || opcode == SpvOpTerminateInvocation);
```

Comment:
The formatting here looks odd.  I don't know if clang-format will fix this up or not.